connect/reports/renderers/xlsx.py

```diff
@@ -19,8 +19,12 @@
 
 @register('xlsx')
 class XLSXRenderer(BaseRenderer):
-    def render(self, data, output_file):
-        start_time = datetime.now(tz=pytz.utc)
+    def render(self, data, output_file, start_time):
```

Comment:
Since you overrides the render method, there is no need to invoke generate_report and generate_summary.
Better, inside the render method to generate both the report and the summary and avoid to write/reopen the excel file.

Reply:
Implemented